cmd/bnscli/cmd_gov.go

```diff
@@ -471,3 +471,43 @@ Reads a transaction from the input and attaches the provided elector address, we
 	_, err = writeTx(output, tx)
 	return nil
 }
+
+func cmdUpdateElectionRule(input io.Reader, output io.Writer, args []string) error {
+	fl := flag.NewFlagSet("", flag.ExitOnError)
+	fl.Usage = func() {
+		fmt.Fprintln(flag.CommandLine.Output(), `
+Creates a new version for an existing election rule. The new version is used for new proposals.
+		`)
+		fl.PrintDefaults()
+	}
+	var (
+		id            = flSeq(fl, "id", "", "The ID of the election rule")
+		durationFl    = fl.Int("voting-period", 0, "Duration in seconds how long the voting period will take place")
```

Comment:
I think it's better to use [`Duration`](https://golang.org/pkg/flag/#Duration) instead of assuming time in seconds.

Reply:
The problem is with anything <1s.like `2h1ms`.  I assume this is not used very often so I did not want to implement any checks to prevent this.

Reply:
You can write in the help text, that the granularity is 1 second and anything smaller is discarded.
I don't think narrowing to 1s should be a problem as humans are not good with such small duration anyway :robot: